app/src/main/java/com/del/delcontainer/utils/Constants.java

```diff
@@ -60,7 +60,7 @@
     // IP addresses and hostnames
     public static final String HTTP_PREFIX          = "http://";
     public static final String HTTPS_PREFIX         = "https://";
-    public static final String DEL_SERVICE_IP       = "192.168.1.9";
+    public static final String DEL_SERVICE_IP       = "192.168.1.174";
```

Comment:
You don't have to submit files with just the IP changes unless we're looking at public addresses. If there were other changes, that would have been fine.

Reply:
Okay I will make a new pull request after pushing new changes.